compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -124,6 +124,12 @@ void GlobalOptimizationOptions::bindOptions(OptionsBinder &binder) {
           "Hoists the results of latent constant expressions into immutable "
           "global initializers for evaluation at program load."),
       llvm::cl::cat(category));
+  binder.opt<int64_t>(
```

Comment:
Wait, you arent using this anywhere yet?

Reply:
There is already a pass option, just no flag. This will overwrite the default value here:
https://github.com/openxla/iree/blob/eec081c2c60da087e3142b9ac2996c35feb4fe4e/compiler/src/iree/compiler/Pipelines/Options.h#L95